tigobot/scripts/pub.coffee

```diff
@@ -2,6 +2,6 @@
 #   Pub!
 
 module.exports = (robot) ->
-  robot.hear /pub/i, (msg) ->
+  robot.hear /^pub\??$/i, (msg) ->
```

Comment:
That would be good yeah, will do when I'm back at a computer

On Wed, 2 Sep 2015 18:33 William Mathewson <notifications@github.com> wrote:

> In tigobot/scripts/pub.coffee
> <https://github.com/rhiaro/tigobot/pull/18#discussion_r38560572>:
>
> > @@ -2,6 +2,6 @@
> >  #   Pub!
> >
> >  module.exports = (robot) ->
> > -  robot.hear /pub/i, (msg) ->
> > +  robot.hear /^pub\??$/i, (msg) ->
>
> Could you change the regex to /^pu+b\??$/i to allow for such things as <Norgg>
> puuuuub
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rhiaro/tigobot/pull/18/files#r38560572>.
>